src/content/docs/fr/guides/deploy/netlify.mdx

```diff
@@ -50,8 +48,8 @@ Si vous préférez installer l'adaptateur manuellement, suivez les deux étapes
     });
     ```
 
-    Vous pouvez également déployer votre projet en utilisant les fonctions Edge de Netlify en ajoutant `edgeMiddleware : true` à la configuration de l'adaptateur Netlify :
-
+    Vous pouvez également déployer le middleware Astro de votre projet en utilisant les fonctions Edge de Netlify en ajoutant `edgeMiddleware : true` à la configuration de l'adaptateur Netlify :
+  
```

Comment:
```suggestion

```

Reply:
For this line, this is on the main version https://github.com/withastro/docs/blob/63e728463e4da50a5dfeb736962ab36bed09f7da/src/content/docs/en/guides/deploy/netlify.mdx?plain=1#L51